ItemSketch throws errors and kicks players on Dedicated Server

In 1.0.2-1.0.4 for 1.10.2

Specifically player.getPositionEyes() does not exist on the server.

```
Caused by: java.lang.NoSuchMethodError: net.minecraft.entity.player.EntityPlayer.func_174824_e(F)Lnet/minecraft/util/math/Vec3d;
	at li.cil.architect.util.PlayerUtils.getLookAtPos(PlayerUtils.java:26) ~[PlayerUtils.class:?]
	at li.cil.architect.common.item.ItemSketch.func_77659_a(ItemSketch.java:111) ~[ItemSketch.class:?]
```
```
Caused by: java.lang.NoSuchMethodError: net.minecraft.entity.EntityLivingBase.func_174824_e(F)Lnet/minecraft/util/math/Vec3d;
	at li.cil.architect.common.item.ItemSketch.onUsingTick(ItemSketch.java:132) ~[ItemSketch.class:?]
```

--- a/src/main/java/li/cil/architect/common/item/ItemSketch.java
+++ b/src/main/java/li/cil/architect/common/item/ItemSketch.java
@@ -138,9 +138,7 @@ public final class ItemSketch extends AbstractItem {
 
     @Override
     public void onUsingTick(final ItemStack stack, final EntityLivingBase player, final int count) {
-        final Vec3d lookAtBase = player.
-                getPositionEyes(1).
-                add(player.getLookVec());
+        final Vec3d lookAtBase = new Vec3d(player.posX, player.posY + player.getEyeHeight(), player.posZ).add(player.getLookVec());
         final Vec3d speedBase = player.
                 getLookVec();
         for (int i = 0; i < 10; i++) {

--- a/src/main/java/li/cil/architect/util/PlayerUtils.java
+++ b/src/main/java/li/cil/architect/util/PlayerUtils.java
@@ -23,7 +23,7 @@ public final class PlayerUtils {
 
     public static BlockPos getLookAtPos(final EntityPlayer player) {
         final Vec3d lookVec = player.getLookVec();
-        final Vec3d eyePos = player.getPositionEyes(1);
+        final Vec3d eyePos = new Vec3d(player.posX, player.posY + player.getEyeHeight(), player.posZ);
         return new BlockPos(eyePos.add(lookVec.scale(freeAimDistance)));
     }
 

